app/src/main/java/com/example/admin/lab_56_57/ActivityForLab57Fragment.java

```diff
@@ -0,0 +1,119 @@
+    private AdapterForRelative adapterForRelative_;
+    private Spinner menuSpinner_;
+    private ArrayAdapter<String> spinnerAdapter_;
+    private final String LINEAR = "Linear Layout";
+    private final String RELATIVE = "Relative Layout";
+
+    public ActivityForLab57Fragment() {
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View rootView = inflater.inflate(R.layout.fragment_activity_for_lab57, container, false);
+
+        // Get values for spinner
+        String[] valuesForSpinner = getResources().getStringArray(R.array.spinner);
+
+        // List of objects for title and description
+        List<AppInfo> values = new ArrayList<AppInfo>();
+
+        PackageManager packageManager = getActivity().getPackageManager();
+        List<ApplicationInfo> packages = packageManager.getInstalledApplications(0);
+
+
+        for (ApplicationInfo packageInfo : packages) {
+            // Size of app
+            double size = Utility.getSize(packageManager,packageInfo);
+            // Getting icon of app
+            Drawable icon = Utility.getIcon(packageManager,packageInfo);
+            // Adding all values to object
+            AppInfo singleItem = new AppInfo(packageInfo.packageName, packageInfo.targetSdkVersion, size,icon);
+            values.add(singleItem);
+        }
+
+
+
+        ListView listView = (ListView) rootView.findViewById(R.id.list_view);
+
+        adapterForRelative_ = new AdapterForRelative(this.getActivity(), values);
+        adapterForLinear_ = new AdapterForLinear(this.getActivity(), values);
+
+        listView.setAdapter(adapterForLinear_);
+
+        spinnerAdapter_ = new ArrayAdapter<String>(
+                getActivity(),
+                R.layout.support_simple_spinner_dropdown_item,
+                valuesForSpinner
+        );
+
+
+        menuSpinner_ = (Spinner) rootView.findViewById(R.id.menu_spinner);
+        menuSpinner_.setAdapter(spinnerAdapter_);
+
+        menuSpinner_.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int pos, long id) {
+                String city = parent.getItemAtPosition(pos).toString();
+                ListView listView = (ListView) getActivity().findViewById(R.id.list_view);
+                if (city.equals(LINEAR)) {
+                    listView.setAdapter(adapterForLinear_);
+                    adapterForRelative_.notifyDataSetChanged();
+                } else if (city.equals(RELATIVE)) {
+                    listView.setAdapter(adapterForRelative_);
+                    adapterForLinear_.notifyDataSetChanged();
+                }
+
+            }
+
+            @Override
+            public void onNothingSelected(AdapterView<?> arg0) {
+                // TODO Auto-generated method stub
```

Comment:
Remove TODO comment if you are not going to TODO something in the future. For instance, rename to ```// Nothing to do```